packages/frontend/src/app.tsx

```diff
@@ -0,0 +1,37 @@
+import { $, component$, useSignal, type Signal } from '@builder.io/qwik';
+
+import { AuthFormWrapper } from './components/AuthFormWrapper';
+import { Dashboard } from './components/Dashboard';
+import type { User } from './types';
+
+export const App = component$(() => {
+  return <AppConsumer />;
+});
+
+const AppConsumer = component$(() => {
+  const form: Signal<'login' | 'register'> = useSignal('login');
+  const darkMode = useSignal(localStorage.getItem('darkMode') === 'true');
```

Comment:
I think we should also abstract out the use of localStarage into own file so if we want to update it to cookies or something else, its a one place file change